guides/common/modules/proc_viewing-content-view-environment-activation-keys.adoc

```diff
@@ -0,0 +1,38 @@
+[id="viewing-content-view-environment-activation-keys"]
+= Viewing content view environment activation keys
+
+You can view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+* Ensure you have activation keys created.
+For more information, see xref:Creating_an_Activation_Key_{context}[].
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.    
+. Select an activation key.
+. Under the *Details* tab, the *Content view environments* are displayed.
+
+[NOTE]
+====
+You cannot reassign a multi-environment activation key's content view environments using the {ProjectWebUI}.
+To change content view environments, update the activation key with Hammer.
+====
```

Comment:
If you decide to remove this module, perhaps this note could go to the assigning module.